src/script/util/util.ts

```diff
@@ -267,6 +267,7 @@ const focusableElementsSelector =
 
 export const preventFocusOutside = (event: KeyboardEvent, parentId: string): void => {
   if (!isTabKey(event)) {
+    event.preventDefault();
     return;
   }
   event.preventDefault();
```

Comment:
how about moving the `event.preventDefault()` before the `if` and avoid this duplicate?

Reply:
duh :upside_down_face: